queries/ruby/textobjects.scm

```diff
@@ -1,28 +1,80 @@
-[
-  (method)
-  (singleton_method)
-] @function.outer
+;; blocks
+(call
+  block: (_) @block.inner) @block.outer
 
+;; calls
+(call
+  method: (_) @call.inner ) @call.outer
+
+;; classes
 [
-  (class)
-  (module)
+  (class
+    name: (constant)
+    superclass: (_)
+    (_) @class.inner)
```

Comment:
`class` could be has no parents.
```suggestion
  (class
    name: (constant)
    superclass: (_)
    (_) @class.inner)
  (class
    name: (constant)
    !superclass
    (_) @class.inner)
```

Reply:
Thanks. this was a bit of oversight on my part